syntax/zsh.nanorc

```diff
@@ -1,5 +1,6 @@
 ## Syntax highlighting for ZSH scripts (initially copied from sh.nanorc)
-syntax "zsh" "\.?zsh[^\.]+?$"
+#syntax "zsh" "\.?zsh[^\.]+?$"
+syntax "zsh" "\.?zsh[^\.]+$" "\.?zlog(in|out)$"
```

Comment:
Same as previous hunk. I also don't use zsh, so I don't know how useful the send part is - `"\.?zlog(in|out)$"`.